File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.py
n,m = map(int, input().split())
basket = list(range(1, n+1))

for _ in range(m):
    i,j = map(int, input().split())
    i-=1
    j-=1
    temp = basket[i]
    basket[i] = basket[j]
    basket[j] = temp

for i in basket:
    print(int(i), end=' ')

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.cc
#include<iostream>
using namespace std;

int main(void){
    ios::sync_with_stdio;
    cin.tie(0);
    int n,m,i,j,k,temp;
    
    cin >> n >> m;
    
	int basket[n];
    
    for(k=0;k<n;k++){
    	basket[k]=k+1;
	}
    
    for(k=0;k<m;k++){
        cin >> i >> j;
        i--; 
        j--;
        temp = basket[i];
        basket[i] = basket[j];
        basket[j] = temp;
    }
    
    for(k=0;k<n;k++){
        cout << basket[k] << " ";
    }
}